src/app/recipe/_common/TopSection.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from "next/link";
+
+import { mockDataRecipe, RecipeAppT10Chefs } from "@/mock";
+
+import { Button } from "@/components/button";
+import { Icon } from "@/components/icon/Icon";
+import { ImageComponent } from "@/components/image";
+
+const favNumber = 1234;
+const chefName = RecipeAppT10Chefs[0].name;
+
+export const TopSection = () => {
+  return (
+    <div className="text-medium">
+      <ImageComponent
+        src={mockDataRecipe[0].image_url1}
+        alt={`${mockDataRecipe[0].title}の画像`}
+        ratio={"1/1"}
+        width={"full"}
+      />
+      <button type="button" className="absolute left-3 top-3 cursor-pointer rounded-full p-1.5">
+        <Link href="/">
+          <Icon type="ArrowLeft" color="white" />
+        </Link>
+      </button>
+      <div className="pt-4">
+        <div className="px-4 pb-3 text-large">
+          <div>シェフのレシピ</div>
+        </div>
+        <div className="px-4">
+          <div>
+            吾輩は猫である。名前はまだ無い。どこで生れたかとんと見当がつかぬ。何でも薄暗いじめじめした所でニャーニャー泣いていた事だけは記憶している。
+          </div>
```

Comment:
なるべく正しいマークアップを意識しましょう！

```suggestion
          <p>
            吾輩は猫である。名前はまだ無い。どこで生れたかとんと見当がつかぬ。何でも薄暗いじめじめした所でニャーニャー泣いていた事だけは記憶している。
          </p>
```